packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -322,15 +318,25 @@ type RestrictedMenuItemProps = {
   dataFences?: TDataFence[];
   children: ReactNode;
 };
+type TLongLivedFlag = {
```

Comment:
We defined our internal flag structure here was we don't yet need a shared type.